LutronLuxmeterIntegration/src/reader.ts

```diff
@@ -0,0 +1,69 @@
+import * as fs from 'fs/promises';
+import { SerialPort } from 'serialport';
+
+export type TResult = { value: number; unit: string };
+
+const bufferSize = 16;
+
+export class LuxMeterReader {
```

Comment:
soubor bych nazval stejne jako tridu LuxMeterReader.ts

Reply:
Měl jsem tendenci soubory pojmenovávat minimalisticky. Ale máš pravdu, taky mě to mate.